Sming/rboot/eagle.rom.addr.v6.ld

```diff
@@ -0,0 +1,347 @@
+PROVIDE ( Cache_Read_Disable = 0x400047f0 );
```

Comment:
I have a small change that works nicely upstream and in Sming that eliminates the need for both eagle.rom.addr.v6.ld and eagle.app.v6.ld in rBoot. There were some odd windows specific problems with the linker script include previously, so before I push it I'll ask @robotiko if he'd mine testing the patch out for me
```
diff --git a/Basic_rBoot/rom0.ld b/Basic_rBoot/rom0.ld
index 8fdb75d..7b2a9f7 100644
--- a/Basic_rBoot/rom0.ld
+++ b/Basic_rBoot/rom0.ld
@@ -216,4 +216,4 @@ SECTIONS
 }
 
 /* get ROM code address */
-INCLUDE "../Sming/rboot/eagle.rom.addr.v6.ld"
+INCLUDE "../ld/eagle.rom.addr.v6.ld"
diff --git a/Sming/rboot/Makefile b/Sming/rboot/Makefile
index b106d0d..30be645 100644
--- a/Sming/rboot/Makefile
+++ b/Sming/rboot/Makefile
@@ -4,6 +4,7 @@
 #
 
 ESPTOOL2 ?= ../esptool2/esptool2
+SDK_BASE ?= /opt/esp-open-sdk/sdk
 
 RBOOT_BUILD_BASE ?= build
 RBOOT_FW_BASE    ?= firmware
@@ -17,8 +18,8 @@ LD := $(addprefix $(XTENSA_BINDIR)/,xtensa-lx106-elf-gcc)
 endif
 
 CFLAGS    = -Os -O3 -Wpointer-arith -Wundef -Werror -Wl,-EL -fno-inline-functions -nostdlib -mlongcalls -mtext-section-literals  -D__ets__ -DICACHE_FLASH
-LDFLAGS   = -nostdlib -Wl,--no-check-sections -u call_user_start -Wl,-static
-LD_SCRIPT = eagle.app.v6.ld
+LDFLAGS   = -nostdlib -Wl,--no-check-sections -u call_user_start -Wl,-static -L $(SDK_BASE)/lib/
+LD_SCRIPT = $(SDK_BASE)/ld/eagle.app.v6.ld
 
 E2_OPTS = -quiet -bin -boot0
 
diff --git a/Sming/rboot/rboot-stage2a.ld b/Sming/rboot/rboot-stage2a.ld
index b263c71..d494212 100644
--- a/Sming/rboot/rboot-stage2a.ld
+++ b/Sming/rboot/rboot-stage2a.ld
@@ -208,4 +208,4 @@ SECTIONS
 }
 
 /* get ROM code address */
-INCLUDE "eagle.rom.addr.v6.ld"
+INCLUDE "../ld/eagle.rom.addr.v6.ld"
```
And delete:
`Sming/Sming/rboot/eagle.app.v6.ld`
`Sming/Sming/rboot/eagle.rom.addr.v6.ld`